recipe/meta.yaml

```diff
@@ -21,8 +21,8 @@ requirements:
   run:
     - python
     - 'configurable-http-proxy >=4.1'
-    - cylc-flow =={{ version }}
-    - cylc-uiserver ==0.2
+    - cylc-flow {{ version }} *_3
+    - cylc-uiserver 0.2 *_2
```

Comment:
Out of interest does `*_3` mean `==3` or `>=3`?

Reply:
I tried `==3`, `3`, but neither worked. Then I found in their docs the section [*Package match specifications*](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications), where they say:

- A match specification is a space-separated string of 1, 2, or 3 parts:
- The first part is always the exact name of the package.
- The second part refers to the version and may contain special characters (BUT... further down)
    - When there are 3 parts, the second part must be the exact version.
- The third part is always the exact build string.

I was confused and thought that the build string would be the build number. But in that documentation they also talk about build number.

So I assumed (didn't find anywhere confirming exactly this) that the build string was the complete value after the version from a conda package, like in `cylc-uiserver`, for the [latest package of Python 3.7](https://anaconda.org/conda-forge/cylc-uiserver/files), we have "cylc-uiserver-0.2-py38_2.tar.bz2". Where I parsed it as:

- cylc-uiserver, as the name of the package
- 0.2, as the package version
- the rest, py38_2, as the build string

Since we don't want to pin at a specific version of Python, I used this example from their docs

>The build string constraint "numpy=1.11.2=*nomkl*" matches the NumPy 1.11.2 packages without MKL but not the normal MKL NumPy 1.11.2 packages.

And used a glob/star to expan after the python version.

So I **think** `*_3` means any build string ending with `_3`.

Reply:
We should document this meaning in our cylc release instructions doc, @kinow

Reply:
Will prepare a PR later today